corehq/apps/api/query_adapters.py

```diff
@@ -59,6 +65,21 @@ def __getitem__(self, item):
         if isinstance(item, slice):
             limit = item.stop - item.start
             result = GroupES().domain(self.domain).sort('name').size(limit).start(item.start).run()
-            return map(Group.wrap, result.hits)
+            groups = [Group.wrap(group) for group in result.hits]
+
+            user_ids = {user_id for group in groups for user_id in group.users}
+            active_user_ids = (
+                UserES().domain(self.domain)
+                .user_ids(user_ids)
```

Comment:
Yeah, while it didn't matter at all for 2.5k user_ids, that's by no means the high end of what we expect, so yeah I think it's foolhardy not to chunk here. Thanks for noticing

Reply:
Oh and this isn't all users in the project by the way—the group API allows for only fetching up to 1000 groups at a time, so this will be however many users fit into 1000 groups. Still, this could get up pretty high if they're all big groups, so I'm going to chunk by like 3000 which I think it can totally handle in one go.

Reply:
Since it's under a flag, I'll follow up on this in another PR